includes/Core/Util/Input.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Input abstraction class.
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Class Input
```

Comment:
The actual class description should be here, e.g. "Class used to filter request input." or something like that.

Reply:
Updated!